src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -0,0 +1,157 @@
+import { mockBlueprintIds } from '../../../../fixtures/blueprints';
+import { usersCreateBlueprintRequest } from '../../../../fixtures/editMode';
+import {
+  blueprintRequest,
+  clickBack,
+  clickNext,
+  enterBlueprintName,
+  getNextButton,
+  interceptBlueprintRequest,
+  interceptEditBlueprintRequest,
+  openAndDismissSaveAndBuildModal,
+  renderEditMode,
+  verifyCancelButton,
+} from '../../wizardTestUtils';
+import {
+  clickRegisterLater,
+  goToRegistrationStep,
+  renderCreateMode,
+} from '../../wizardTestUtils';
+
+let router: RemixRouter | undefined = undefined;
+
+const goToUsersStep = async () => {
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+};
+
+const goToReviewStep = async () => {
+  await clickNext(); // Timezone
+  await clickNext(); // Locale
+  await clickNext(); // Hostname
+  await clickNext(); // Kernel
+  await clickNext(); // First boot
+  await clickNext(); // Details
+  await enterBlueprintName();
+  await clickNext(); // Review
+};
+
+const addValidUser = async () => {
+  const user = userEvent.setup();
+  const addUser = await screen.findByRole('button', { name: /add a user/i });
+  expect(addUser).toBeEnabled();
+  await waitFor(() => user.click(addUser));
+  const enterUserName = screen.getByRole('textbox', {
+    name: /blueprint user name/i,
+  });
+  const nextButton = await getNextButton();
+  await waitFor(() => user.type(enterUserName, 'best'));
+  await waitFor(() => expect(enterUserName).toHaveValue('best'));
+  await waitFor(() => expect(nextButton).toBeEnabled());
+};
+
+describe('Step Users', () => {
+  beforeEach(async () => {
+    vi.clearAllMocks();
+    router = undefined;
+  });
+
+  test('without adding user loads timezone', async () => {
+    const user = userEvent.setup();
+    await renderCreateMode();
+    await goToRegistrationStep();
+    await clickRegisterLater();
+    await goToUsersStep();
+    const addUser = await screen.findByRole('button', { name: /add a user/i });
+    expect(addUser).toBeEnabled();
+    const nextButton = await getNextButton();
+    expect(nextButton).toBeEnabled();
+    await waitFor(() => user.click(nextButton));
+    await screen.findByText('Select a timezone for your image.');
+  });
+
+  test('clicking Back loads Additional packages', async () => {
+    await renderCreateMode();
+    await goToRegistrationStep();
+    await clickRegisterLater();
+    await goToUsersStep();
+    const addUser = await screen.findByRole('button', { name: /add a user/i });
+    expect(addUser).toBeEnabled();
+    await clickBack();
+    await screen.findByRole('heading', { name: 'Additional packages' });
+  });
+
+  test('clicking Cancel loads landing page', async () => {
+    await renderCreateMode();
+    await goToUsersStep();
+    await verifyCancelButton(router);
+  });
+
+  test('with valid name', async () => {
```

Comment:
This test is identical to the 'with valid name and password' test that appears in the 'User request generated correctly' block up to (and including) goToReviewStep().

Keep in mind that every test takes time to run, and the faster the test suite runs the better the developer experience. We want coverage to be high, but we also want to avoid duplication.

We will want to add tests that check the behavior for an _invalid_ name, of course, once we add validation.